cloud_formation.yaml

```diff
@@ -0,0 +1,653 @@
+      z: 1
+      embeds: []
+    e6ba90e0-7d27-4bee-ad7f-643e3e678a4d:
+      size:
+        width: 60
+        height: 60
+      position:
+        x: 510
+        'y': 120
+      z: 1
+      embeds: []
+    d47edd72-2c89-4402-8d9d-b491fc9a9391:
+      size:
+        width: 60
+        height: 60
+      position:
+        x: 430
+        'y': 300
+      z: 1
+      embeds: []
+    f8454eab-09e5-492b-b18d-3cd73e878ec7:
+      size:
+        width: 60
+        height: 60
+      position:
+        x: 60
+        'y': 270
+      z: 1
+      embeds: []
+    09ff0abd-3273-42ed-8bde-de6ef96598e9:
+      size:
+        width: 60
+        height: 60
+      position:
+        x: -40
+        'y': 250
+      z: 1
+      embeds: []
+    e8f7da01-fcb2-4c3f-b331-0df7d593927d:
+      size:
+        width: 60
+        height: 60
+      position:
+        x: 270
+        'y': 300
+      z: 1
+      embeds: []
+    e153092a-2a0b-4490-85df-2e03b367696b:
+      size:
+        width: 60
+        height: 60
+      position:
+        x: 330
+        'y': 240
+      z: 1
+      embeds: []
+    23fe9e8e-1ce3-4fb7-b272-431946d1c309:
+      size:
+        width: 60
+        height: 60
+      position:
+        x: 390
+        'y': 30
+      z: 1
+      embeds: []
+    78db63b4-6960-4fe9-87fc-47fd6ffeaad0:
+      size:
+        width: 60
+        height: 60
+      position:
+        x: -250
+        'y': 40
+      z: 1
+      embeds: []
+      isassociatedwith:
+        - 690cb47e-5923-4e28-883c-a11440f4adc4
+    76a9ea71-1449-4142-a6ad-505f57484071:
+      size:
+        width: 60
+        height: 60
+      position:
+        x: 810
+        'y': 90
+      z: 1
+      embeds: []
+    1ae3457b-36e3-48fd-8fd7-b309db18b087:
+      size:
+        width: 60
+        height: 60
+      position:
+        x: 910
+        'y': 210
+      z: 1
+      embeds: []
+      isassociatedwith:
+        - 76a9ea71-1449-4142-a6ad-505f57484071
+    2cc6e775-6571-4763-ae4c-3bb188cbcc39:
+      size:
+        width: 60
+        height: 60
+      position:
+        x: 120
+        'y': 90
+      z: 1
+      embeds: []
+    f5718e5c-4739-4f0b-8f1f-06c1cac973ad:
+      size:
+        width: 60
+        height: 60
+      position:
+        x: 150
+        'y': 150
+      z: 1
+      embeds: []
+    8d3e8115-df67-406c-a7bb-983f69295530:
+      size:
+        width: 60
+        height: 60
+      position:
+        x: 30
+        'y': 30
+      z: 1
+      embeds: []
+    ad2d2fb6-1163-4706-84f2-bd0530148987:
+      size:
+        width: 60
+        height: 60
+      position:
+        x: 30
+        'y': 150
+      z: 1
+      embeds: []
+    911e338a-e83d-49c7-98ab-680580006084:
+      size:
+        width: 60
+        height: 60
+      position:
+        x: 30
+        'y': 90
+      z: 1
+      embeds: []
+    3c90b7a5-2070-44f0-9e2d-18547e7d7998:
+      size:
+        width: 60
+        height: 60
+      position:
+        x: -252.6180668549284
+        'y': 150.01180493985788
+      z: 0
```

Comment:
Get rid of all the Designer gunk in this section, and the metadata related to it in the resources.